src/js/components/Select/SelectContainer.js

```diff
@@ -260,23 +263,42 @@ const SelectContainer = forwardRef(
           setActiveIndex(nextActiveIndex);
           setKeyboardNavigation(true);
         }
+        if (
+          clear &&
+          clear.position === 'bottom' &&
+          clearButton &&
+          nextActiveIndex >= options.length
+        ) {
+          setActiveIndex(options.length + 1);
```

Comment:
```suggestion
          setActiveIndex(options.length);
          setFocusWithoutScroll(clearButton);
```

Reply:
yes i think `option.length` makes sense

as far as the `setFocusWithoutScroll`: i had this code implemented and i thought it would have worked but it still doesn't set the focus on the button. this was the code snippet i was still struggling with, as i thought doing `setFocusWithoutScroll` would handle it similarly to when the button is at the beginning.

i'll make the suggested changes from above and push so it's updated.